src/backends/mod.rs

```diff
@@ -54,6 +55,12 @@ pub fn execute_command<'a>(
                 }
             }
         }
+        CommandSpec::Clean => {
```

Comment:
Should we also remove `Lingo.toml` here?

Reply:
no